src/plugins/github/README.md

```diff
@@ -0,0 +1,175 @@
+add much to the plugin at the moment.
+
+[this commit]: https://github.com/sourcecred/sourcecred/commit/94b04541514b991c304616aadfcb417a19871e82
+
+### User
+
+A GitHub user account, e.g. [@decentralion]. User accounts do not mint cred, so
+setting a node weight would have no effect. Using the identity plugin, it's
+possible to "collapse" user nodes with other identity nodes into a single,
+canonical identity. For example, if a contributor had a GitHub user account and
+a Discourse account, then the identity plugin can collapse those identities
+together.
+
+Users are connected to posts they author, to posts they react to, and to posts that
+mention them.
+
+[@decentralion]: https://github.com/decentralion
+
+### Bot
+
+A GitHub user account that has been explicitly marked as a bot, via inclusion
+in [bots.js]. This is useful so that we can filter out bot accounts from
+receiving grain or showing up in the cred rankings.
+
+Bots have the same connections as users.
+
+[bots.js]: https://github.com/sourcecred/sourcecred/blob/master/src/plugins/github/bots.js
+
+
+## Edges
+
+### Authors
+
+An authors edge connects an author (i.e. a user or bot) to a post (i.e. a pull,
+issue, comment, or review). If the post contains the text "paired with
+@other-author", then from SourceCred's perspective, that post will have
+multiple authors, all of whom receive an equal share of the cred.
+
+The following are all valid examples of using "paired with":
+
+> paired with @wchargin
+>
+> paired-with @wchargin
+>
+> paired with: @wchargin
+>
+> Paired with @wchargin
+>
+> Paired With @wchargin
+>
+> Paired With: @wchargin
+>
+> Paired-With: @wchargin
+>
+
+### References
+
+A references edge connects a post (i.e. a pull, issue, comment, or review) to
+another referencable node (i.e. a node that corresponds to a specific url on
+GitHub).
+
+If the reference is pointing to a user, we call it a "mention", but
+from SourceCred's perspective it's the same kind of edge.
+
+### Reacts
+
+A react edge connects a user (or bot) to a pull, issue, or comment. There are subtypes
+of reaction edges corresponding to the type of reaction; currently we support thumbs-up,
+heart, and hooray emojis. In the future, we might reify the reactions as nodes, so as to
+support reaction-minted cred, in the style of [like-minted cred].
+
+[like-minted cred]: https://discourse.sourcecred.io/t/minting-discourse-cred-on-likes-not-posts/603
+
+### Has Parent
+
+A has-parent edge connects a "child" node to its "parent" node. Here's a table
+summarizing these relationships:
+
+| Child | Parent |
+| --- | --- |
+| Issue | Repository |
+| Issue Comment | Issue |
+| Pull Request | Repository |
+| Pull Comment | Pull Request |
+| Pull Request Review | Pull Request |
+| Pull Request Review Comment | Pull Request Review |
+
+### Merged As
+
+A merged-as edge connects a pull request to the commit that it merged, assuming
+the pull request merged.
```

Comment:
that the pull request _was_ merged (unless the pull request can merge itself :P)

Reply:
Fixed.